plugins/source/homebrew/README.md

```diff
@@ -0,0 +1,7 @@
+# Homebrew Source Plugin
+
+The Homebrew Source plugin for CloudQuery extracts configuration from the [Homebrew Analytics API](https://formulae.brew.sh/analytics/) and loads it into any supported CloudQuery destination (e.g. PostgreSQL, BigQuery, Snowflake, and [more](https://www.cloudquery.io/docs/plugins/destinations/overview)).
```

Comment:
```suggestion
The Homebrew Source plugin for CloudQuery extracts configuration from the [Homebrew Analytics API](https://formulae.brew.sh/analytics/) and loads it into any supported CloudQuery destination (e.g. PostgreSQL, BigQuery, Snowflake, and [more](/docs/plugins/destinations/overview)).
```

Internal docs should have relative(ish) links?

Reply:
Hmm no, this is the readme, it should be absolute